src/Serialization/main.jl

```diff
@@ -658,7 +709,8 @@ function load(io::IO; params::Any = nothing, type::Any = nothing,
   end
 end
 
-function load(filename::String; params::Any = nothing, type::Any = nothing)
+function load(filename::String; params::Any = nothing,
+              type::Any = nothing, with_attrs::Bool=false)
```

Comment:
```suggestion
              type::Any = nothing, with_attrs::Bool=true)
```
default here should be the same as for the `load(::IO)` dispatch